tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/CIVisibilityTests.cs

```diff
@@ -0,0 +1,29 @@
+// <copyright file="CIVisibilityTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI
+{
+    public class CIVisibilityTests
+    {
+        [Theory]
+        [InlineData("https://github.com/DataDog/dd-trace-dotnet.git", "dd-trace-dotnet")]
+        [InlineData("git@github.com:DataDog/dd-trace-dotnet.git", "dd-trace-dotnet")]
+        [InlineData("ssh://user@host.xz:port/path/to/repo.git", "repo")]
+        [InlineData("ssh://user@host.xz/~/path/to/repo.git", "repo")]
+        [InlineData("rsync://host.xz/path/to/repo.git", "repo")]
+        [InlineData("git://host.xz/~user/path/to/repo.git", "repo")]
+        [InlineData("file://~/path/to/repo.git", "repo")]
+        [InlineData("/path/to/repo.git/", "repo")]
+        [InlineData("user@host.xz:~user/path/to/repo.git/", "repo")]
+        [InlineData("ssh://login@server.com:12345/absolute/path/to/repository", "repository")]
+        [InlineData("ssh://login@server.com:12345/repository.git", "repository")]
```

Comment:
Could we add a few more test scenarios, e.g. edge cases, and windows file paths? (I've guessed at the expected values, have confirmed they're expected or correct!)
```suggestion
        [InlineData("ssh://login@server.com:12345/repository.git", "repository")]
        [InlineData("repo.git", "repo")]
        [InlineData("./repo", "repo")]
        [InlineData(@".\repo", "repo")]
        [InlineData(@"\\wsl$\path\to\repo", "repo")]
        [InlineData(@"C:\path\to\repo", "repo")]
        [InlineData("", "")]
        [InlineData("%^&*", "")]       
```

Reply:
added in 7997f88 (also fixes the regex to handle those cases 😺  )